go.sum

```diff
@@ -301,6 +301,8 @@ github.com/onsi/gomega v1.9.0 h1:R1uwffexN6Pr340GtYRIdZmAiN4J+iw6WG4wog1DUXg=
 github.com/onsi/gomega v1.9.0/go.mod h1:Ho0h+IUsWyvy1OpqCwxlQ/21gkhVunqlU8fDGcoTdcA=
 github.com/open-policy-agent/frameworks/constraint v0.0.0-20200919011008-0a87f65d4444 h1:85VGMMAldK+pnCcttbD/0E0sBFSqBqrLJW4BYqBFheY=
 github.com/open-policy-agent/frameworks/constraint v0.0.0-20200919011008-0a87f65d4444/go.mod h1:Dr3QxvH+NTQcPPZWSt1ueNOsxW4VwgUltaLL7Ttnrac=
```

Comment:
these should be removed by `go mod tidy`?

Reply:
Right! Updated the pr, thanks for pointing it out.